src/Symfony/Component/Runtime/SymfonyRuntime.php

```diff
@@ -142,14 +151,16 @@ public function getRunner(?object $application): RunnerInterface
             }
 
             set_time_limit(0);
-            $defaultEnv = null === $this->env ? ($_SERVER['APP_ENV'] ?? 'dev') : null;
+            $defaultEnv = !isset($this->options['env']) ? $_SERVER['APP_ENV'] : null;
+            $input = $this->input ?? $this->input = new ArgvInput();
             $output = $this->output ?? $this->output = new ConsoleOutput();
 
-            return new ConsoleApplicationRunner($application, $defaultEnv, $this->getInput(), $output);
+            return new ConsoleApplicationRunner($application, $defaultEnv, $this->input, $output);
```

Comment:
This is exactly the same. But currently $input is not used
```suggestion
            return new ConsoleApplicationRunner($application, $defaultEnv, $input, $output);
```